src/components/LineGraph.tsx

```diff
@@ -153,7 +153,7 @@ export const LineGraph: React.FunctionComponent<LineGraphProps> = ({ line: selec
                 max:
                   range === 'past'
                     ? dayjs(new Date()).toISOString()
-                    : dayjs(new Date()).add(2, 'months').toISOString(),
+                    : dayjs(new Date()).add(11, 'months').toISOString(),
```

Comment:
I think this was set to 2 months instead of 3 for a reason. Can't remember why, but can we set this to n-1 (10 months) just in case